test/hotspot/jtreg/serviceability/jvmti/vthread/FollowReferences/VThreadStackRefTest.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @requires vm.jvmti
+ * @requires vm.continuations
+ * @enablePreview
+ * @run main/othervm/native
+ *      -Djava.util.concurrent.ForkJoinPool.common.parallelism=1
+ *      -agentlib:VThreadStackRefTest
+ *      VThreadStackRefTest
+ */
+import java.util.List;
+import java.util.ArrayList;
+import java.util.stream.Stream;
+import java.util.concurrent.CountDownLatch;
+
+public class VThreadStackRefTest {
+
+    static volatile boolean timeToStop = false;
+    static int i = -1;
+
+    public static void main(String[] args) throws InterruptedException {
+        CountDownLatch dumpedLatch = new CountDownLatch(1);
+        Thread vthreadMounted = Thread.ofVirtual().start(() -> {
+            Object referenced = new VThreadMountedReferenced();
+            System.out.println(referenced.getClass());
+            while (!timeToStop) {
+                if (++i == 10000) {
+                    i = 0;
+                }
+            }
+            await(dumpedLatch);
```

Comment:
await() seems unnecessary given the use the !timeToStop flag.

Reply:
Correct. Fixed.